tinygrad/runtime/support/elf.py

```diff
@@ -35,4 +35,7 @@ def _to_carray(sh, ctype): return (ctype * (sh.header.sh_size // sh.header.sh_en
     rels = [(r.r_offset, symtab[libc.ELF64_R_SYM(r.r_info)], libc.ELF64_R_TYPE(r.r_info), getattr(r, "r_addend", 0)) for r in c_rels]
     relocs += [(target_image_off + roff, sections[sym.st_shndx].header.sh_addr + sym.st_value, rtype, raddend) for roff, sym, rtype, raddend in rels]
 
-  return memoryview(image), sections, relocs
+  # Exports
+  exports = {_strtab(sections[sl].content, sym.st_name):sections[sym.st_shndx].header.sh_addr + sym.st_value for sym in symtab if sym.st_info == 0x12}
```

Comment:
st_info 0x12 is ELF_ST_INFO(STB_GLOBAL, STT_FUNC)

Reply:
Can you get that from libc?